app/app.js

```diff
@@ -126,7 +126,30 @@ function (app, $, _, Backbone, Bootstrap, Helpers, Utils, FauxtonAPI, Couchdb) {
           type: 'REMOVE_NAVBAR_LINK',
           link: link
       });
+    },
+
+    isRunningOnBackdoorPort: function () {
```

Comment:
for now it is the feature detection if we run on a backdoor port or not

we'll have to change fauxton anyway if we get `_config` back - if ...